packages/tradershub/src/App.tsx

```diff
@@ -12,7 +13,9 @@ const App = () => (
             <BreakpointProvider>
                 <Provider.CFDProvider>
                     <Provider.ModalProvider>
-                        <AppContent />
+                        <SignupWizardProvider>
```

Comment:
@shahzaib-deriv do you need this because of hiding/showing the modal?

Reply:
Yes @mahdiyeh-deriv this is required for the flow to be available from any point in Tradershub. Keeping it here makes it available to be triggered from anywhere and the content inside is not mounted unless we open the modal like a dynamic component